starter-code/src/clue.js

```diff
@@ -1,94 +1,258 @@
+    last_name: "Plum",
+    color: "purple",
+    description: "Billionare video game designer",
+    age: 22,
+    image: "https://metrouk2.files.wordpress.com/2016/07/professor-plum.jpg",
+    occupation: "Designer"
+}
+
+const missScarlet = {
+    first_name: "Kasandra",
+    last_name: "Scarlet",
+    color: "red",
+    description: "She is an A-list movie star with a dark past",
+    age: 31,
+    image: "https://metrouk2.files.wordpress.com/2016/07/miss-scarlett.jpg",
+    occupation: "Actor"
+}
+
+const mrsPeacock = {
+    first_name: "Eleanor",
+    last_name: "Peacock",
+    color: "blue",
+    description: "She is from a wealthy family and uses her status and money to earn popularity",
+    age: 36,
+    image: "https://metrouk2.files.wordpress.com/2016/07/mrs-peacock.jpg",
+    occupation: "Socialité"
+}
+
+const mrMustard = {
+    first_name: "Jack",
+    last_name: "Mustard",
+    color: "yellow",
+    description: "He is a former football player who tries to get by on his former glory",
+    age: 62,
+    image: "https://metrouk2.files.wordpress.com/2016/07/colonel-mustard.jpg",
+    occupation: "Retired Football Player"
+}
 
 // Weapons
+const rope = {
+    name: "rope",
+    weight: 10
+}
+
+const knife = {
+    name: "knife",
+    weight: 8
+}
+
+const candlestick = {
+    name: "candlestick",
+    weight: 2
+}
+
+const dumbell = {
+    name: "dumbbell",
+    weight: 30
+}
+
+const poison = {
+    name: "poison",
+    weight: 2
+}
 
-name: rope        ---   weight: 10
-name: knife       ---   weight: 8
-name: candlestick ---   weight: 2
-name: dumbbell    ---   weight: 30
-name: poison      ---   weight: 2
-name: axe         ---   weight: 15
-name: bat         ---   weight: 13
-name: trophy      ---   weight: 25
-name: pistol      ---   weight: 20
+const axe = {
+    name: "axe",
+    weight: 15
+}
+
+const bat = {
+    name: "bat",
+    weight: 13
+}
+
+const trophy = {
+    name: "trophy",
+    weight: 25
+}
+
+const pistol = {
+    name: "pistol",
+    weight: 20
+}
 
 // Rooms
-name: Dinning Room
-name: Conservatory
-name: Kitchen
-name: Study
-name: Library
-name: Billiard Room
-name: Lounge
-name: Ballroom
-name: Hall
-name: Spa
-name: Living Room
-name: Observatory
-name: Theater
-name: Guest House
-name: Patio
-*/
+const diningRoom = {
+    name: "Dinning Room"
+}
+
+const conservatory = {
+    name: "Conservatory"
+}
+
+const kitchen = {
+    name: "Kitchen"
+}
+
+const study = {
+    name: "Study"
+}
+
+const library = {
+    name: "Library"
+}
+
+const billiardRoom = {
+    name: "Billiard Room"
+}
+
+const lounge = {
+    name: "Lounge"
+}
+
+const ballroom = {
+    name: "Ballroom"
+}
+
+const hall = {
+    name: "Hall"
+}
+
+const spa = {
+    name: "Spa"
+}
+
+const livingRoom = {
+    name: "Living Room"
+}
+
+const observatory = {
+    name: "Observatory"
+}
+
+const theater = {
+    name: "Theater"
+}
+
+const guestHouse = {
+    name: "Guest House"
+}
+
+const patio = {
+    name: "Patio"
+}
 
```

Comment:
I think I would have declared the rooms a bit differently.  I think naming the Room objects is a little unnecessary because they each have a single property, which is their name.    I would declare the rooms directly in the array: 
```
var rooms = [
{ name: "Guest house" },
{ name: "Patio },
{ name: "Theater" },
{ name: "Observatory" } ]
```
This way you don't have to use the `Array.push` method heaps of times.  Also, you might be interested in checking out constructors.  You can use a constructor function to initialize instances of objects (e.g. Room, Character, Weapon) . Like so:

```
function Room(name) {
  this.name = name;
}
var rooms = [
new Room("Patio"),
new Room("Guest house"),  ...
]
```
But in reality.. totally fine!! I only say this because you might be interested.